Updated ISBN handling

ASimpleFox opened the pull request:
Updated ISBN handling to take isbn_13 whenever possible, and isbn_10 otherwise

Change log:
- Updated the import script to prefer isbn_13 over isbn_10 whenever possible
- Fancied up the script a wee bit!

Note:
- It seems like I have the code checking for the ISBN to be exactly numbers (and also returning the final ISBN as an integer), should I return strings instead? It would keep the leading 0s that occur in some isbn_10 numbers.

Closes #147

ElliotSchumacher commented:
Yes, would you please keep them as strings instead of numbers to retain their leading zeros. I've changed the database to expect `char(13)` for the `ISBN` column so strings should work just fine.

ASimpleFox commented:
Reposting with modifications:

Updated ISBN handling to take isbn_13 whenever possible, and isbn_10 otherwise

Change log:
- Updated the import script to prefer isbn_13 over isbn_10 whenever possible
- Fancied up the script a wee bit!

Note:
- It was mentioned on the merge request comments that the database system has been modified to accommodate `char(13)` instead of `int`, so that change was not implemented on this local branch. Recommending testing this branch after database changes are merged in, or test with the hosted database (assuming that one is the most updated!)
- Tl;dr, changes were modified but not extensively tested as a whole. Please test!

Closes #147

ElliotSchumacher reviewed (approved):
This looks good. I'll go through and test this with my pull request
